packages/reusable-blocks/src/private-apis.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * WordPress dependencies
+ */
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+
+export const { lock, unlock } =
+	__dangerousOptInToUnstableAPIsOnlyForCoreModules(
+		'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.',
+		'@wordpress/edit-site'
```

Comment:
```suggestion
		'@wordpress/reusable-blocks'
```

Reply:
It seems like that name is supposed to be the package you are importing from so switched it to `block-editor`. If I set it to `reusable-blocks` it throws an error about private APIs only be allowed in core packages

Reply:
switching it to `@wordpress/reusable-blocks` causes an error :
```
Uncaught Error: You tried to opt-in to unstable APIs as module "@wordpress/reusable-blocks". This feature is only for JavaScript modules shipped with WordPress core.
```
weird that it doesn't see `@wordpress/reusable-blocks` as a core package. I will take a look at the `__dangerousOptInToUnstableAPIsOnlyForCoreModules` and try and work out why

Reply:
Does it have to be in `CORE_MODULES_USING_PRIVATE_APIS`?

https://github.com/WordPress/gutenberg/blob/33bf984897683adfc4582a29adb079cf6ee0ab59/packages/private-apis/src/implementation.js#L12-L12

Reply:
yep, sure did, sorted.